include/RAJA/RAJA.hpp

```diff
@@ -111,6 +111,13 @@
 #include "RAJA/util/PermutedLayout.hpp"
 #include "RAJA/util/View.hpp"
 
+
+//
+// Atomic operations support
+//
+#include "RAJA/util/atomic.hpp"
```

Comment:
I think `util` should be `pattern` since we have other patterns (forall, scan, reduce) for each policy. Placing in `util` doesn't quite fit this existing model